deepchem/models/tests/test_weavemodel_pytorch.py

```diff
@@ -340,3 +295,77 @@ def test_weave_model_reload():
     original_predict = original_model.predict(dataset)
     reloaded_predict = reloaded_model.predict(dataset)
     assert np.all(original_predict == reloaded_predict)
+
+
+@pytest.mark.torch
+def test_weave_singletask_classification_overfit():
+    """Test weave model overfits tiny data."""
+    # np.random.seed(123)
+    # torch.manual_seed(123)
```

Comment:
I think seed should be turned on for this test.

Reply:
Done